app/src/main/java/info/nightscout/androidaps/utils/stats/TddCalculator.kt

```diff
@@ -121,14 +123,14 @@ class TddCalculator @Inject constructor(
             tdd.carbs += t.amount
         }
         val calculationStep = T.mins(5).msecs()
-        for (t in startTime until endTime step calculationStep) {
-            val tbr = iobCobCalculator.getTempBasalIncludingConvertedExtended(t)
```

Comment:
this call is done inside getBasalData in case of cache miss, so it is fair replacement